server/tests/mocks/ingredientsMock.js

```diff
@@ -1,28 +1,72 @@
 class IngredientMock{
 	constructor(){
 		this.ingredientName = "Tomato";
-        this.ingredientAmount = 2;
-        this.ingredientUnit = "whole";
-        this.recipeID = 1;
+	        this.ingredientAmount = 2;
```

Comment:
I think initially, this constructor should only have 1 property "this.ingredients = []", and additional ingredients can be created via the mocked bulkCreate(). The setup was like this in the beginning because addIngredients() was not implemented before.